File: hexify.py
import base64
import binascii
import string

input_string = "This is a sample string to be secured"

UNICODE_PREFIX = base64.b64decode(b'XA==').decode(encoding="ascii") + string.ascii_letters[20] + string.digits[:1] * 2


def hexlify():
    str_to_hex = UNICODE_PREFIX + UNICODE_PREFIX.join(binascii.hexlify(data=input_string.encode(encoding="utf-8"),
                                                                       sep="-").decode(encoding="utf-8").split(sep="-"))
    hex_to_str = bytes(str_to_hex, "utf-8").decode(encoding="unicode_escape")
    assert hex_to_str == input_string


if __name__ == '__main__':
    hexlify()

File: hexify.js
function ConvertStringToHex(str) {
    var arr = [];
        for (var i = 0; i < str.length; i++) {
            arr[i] = ("00" + str.charCodeAt(i).toString(16)).slice(-4);
        }
    return "\\u" + arr.join("\\u");
}

function ConvertHexToString(Hex) {
    var hex = Hex.toString();
    var str = '';
    for (var i = 0; i < hex.length; i += 2)
        str += String.fromCharCode(parseInt(hex.substr(i, 2), 16));
    return str;
}

var str = "This is a sample string to be secured";

var str_to_hex = ConvertStringToHex(str)
console.log(str_to_hex);

var hex_to_str = ConvertHexToString(str_to_hex)
console.log(hex_to_str);
